Beam/resources/assets/js/components/IotaService.vue

```diff
@@ -1,199 +1,307 @@
 <script type="text/javascript">
-    import Axios from "axios"
-    import EventHub from "./EventHub"
-
-    export default {
-        name: 'iota-service',
-        props: {
-            articleIds: {
-                type: Array,
-                required: true,
-            },
-            pageviewMinuteRanges: {
-                type: Array,
-                default: function() {
-                    return [
-                        {minutes: 15, label: "15m", order: 0},
-                        {minutes: undefined, label: "Total", order: 1},
-                    ];
-                },
+import Axios from "axios";
+import EventHub from "./EventHub";
+
+export default {
+  name: "iota-service",
+  props: {
+    articleIds: {
+      type: Array,
+      required: true
+    },
+    articleDetailId: {
+      type: String,
+      required: false
+    },
+    pageviewMinuteRanges: {
+      type: Array,
+      default: function() {
+        return [
+          { minutes: 15, label: "15m", order: 0 },
+          { minutes: undefined, label: "Total", order: 1 }
+        ];
+      }
+    },
+    variantsMinuteRanges: {
+      type: Array,
+      default: function() {
+        return [
+          { minutes: 15, label: "15m", order: 0 },
+          { minutes: undefined, label: "Total", order: 0 }
```

Comment:
Considering `pageviewMinuteRanges`, you probably wanted to use `order: 1` here

Reply:
This was probably there from some time ago, because I just formatted it. 
OK, changed order to 1.